src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/commands.py

```diff
@@ -19,23 +20,43 @@ def load_command_table(self, _):
         operations_tmpl='azure.mgmt.redis.operations.patch_schedules_operations#PatchSchedulesOperations.{}',
         client_factory=cf_patch_schedules)
 
+    redis_firewall_rules = CliCommandType(
+        operations_tmpl='azure.mgmt.redis.operations.firewall_rules_operations#FirewallRulesOperations.{}',
+        client_factory=cf_firewall_rule)
+
+    redis_linked_server = CliCommandType(
+        operations_tmpl='azure.mgmt.redis.operations.linked_server_operations#LinkedServerOperations.{}',
+        client_factory=cf_linked_server)
+
     with self.command_group('redis', redis_sdk) as g:
         g.custom_command('create', 'cli_redis_create', client_factory=cf_redis,
                          exception_handler=wrong_vmsize_argument_exception_handler)
         g.command('delete', 'delete')
         g.custom_command('export', 'cli_redis_export')
         g.command('force-reboot', 'force_reboot')
         g.custom_command('import-method', 'cli_redis_import_method')
+        g.command('import', 'import_data')
         g.command('list', 'list_by_resource_group')
         g.command('list-all', 'list')
```

Comment:
Not related to this PR, but there shouldn't be a "list-all" command.  The list command should follow the conventions of other list commands in the CLI by folding the list by subscription/resource-group functionality into a single command.